src/Sentry/Laravel/Tracing/Storage/TracingFilesystem.php

```diff
@@ -0,0 +1,183 @@
+        $context->setData(array_merge($data, [
+            'disk' => $this->disk,
+            'driver' => $this->driver,
+        ]));
+        $context->setDescription($description);
+
+        return trace(function () use ($method, $args) {
+            return $this->filesystem->{$method}(...$args);
+        }, $context);
+    }
+
+    /** @see \Illuminate\Filesystem\FilesystemAdapter::assertExists() */
+    public function assertExists($path, $content = null)
+    {
+        return $this->withTracing(__FUNCTION__, func_get_args(), $path, compact('path'));
+    }
+
+    /** @see \Illuminate\Filesystem\FilesystemAdapter::assertMissing() */
+    public function assertMissing($path)
+    {
+        return $this->withTracing(__FUNCTION__, func_get_args(), $path, compact('path'));
+    }
+
+    /** @see \Illuminate\Filesystem\FilesystemAdapter::assertDirectoryEmpty() */
+    public function assertDirectoryEmpty($path)
+    {
+        return $this->withTracing(__FUNCTION__, func_get_args(), $path, compact('path'));
+    }
+
+    public function exists($path)
+    {
+        return $this->withTracing(__FUNCTION__, func_get_args(), $path, compact('path'));
+    }
+
+    public function get($path)
+    {
+        return $this->withTracing(__FUNCTION__, func_get_args(), $path, compact('path'));
+    }
+
+    public function readStream($path)
+    {
+        return $this->withTracing(__FUNCTION__, func_get_args(), $path, compact('path'));
+    }
+
+    public function put($path, $contents, $options = [])
+    {
+        return $this->withTracing(__FUNCTION__, func_get_args(), $path, compact('path', 'options'));
+    }
+
+    public function writeStream($path, $resource, array $options = [])
+    {
+        return $this->withTracing(__FUNCTION__, func_get_args(), $path, compact('path', 'options'));
+    }
+
+    public function getVisibility($path)
+    {
+        return $this->withTracing(__FUNCTION__, func_get_args(), $path, compact('path'));
+    }
+
+    public function setVisibility($path, $visibility)
+    {
+        return $this->withTracing(__FUNCTION__, func_get_args(), $path, compact('path', 'visibility'));
+    }
+
+    public function prepend($path, $data)
+    {
+        return $this->withTracing(__FUNCTION__, func_get_args(), $path, compact('path'));
+    }
+
+    public function append($path, $data)
+    {
+        return $this->withTracing(__FUNCTION__, func_get_args(), $path, compact('path'));
+    }
+
+    public function delete($paths)
+    {
+        if (is_array($paths)) {
+            $data = compact('paths');
+            $description = sprintf('%s paths', count($paths));
+        } else {
+            $data = ['path' => $paths];
+            $description = $paths;
+        }
+
+        return $this->withTracing(__FUNCTION__, func_get_args(), $description, $data);
+    }
+
+    public function copy($from, $to)
+    {
+        return $this->withTracing(__FUNCTION__, func_get_args(), $from, compact('from', 'to'));
```

Comment:
```suggestion
        return $this->withTracing(__FUNCTION__, func_get_args(), "From {$from} to {$to}", compact('from', 'to'));
```

Same with `move`, perhaps adding quotes around the paths

Reply:
On the one hand, yes, on the other it makes the description very long potentially... 

@cleptric WDYT?

Reply:
Right now, this looks like this for me.

![Screenshot 2023-07-25 at 20 00 20](https://github.com/getsentry/sentry-laravel/assets/6617432/93df983d-ef0c-4395-a545-b2b1d84568a6)